portal-kernel/src/com/liferay/asset/kernel/configuration/provider/AssetCategoryConfigurationProviderUtil.java

```diff
@@ -47,6 +47,6 @@ public static boolean isSearchHierarchical(long companyId) {
 			ServiceProxyFactory.newServiceTrackedInstance(
 				AssetCategoryConfigurationProvider.class,
 				AssetCategoryConfigurationProviderUtil.class,
-				"_assetCategoryConfigurationProvider", true);
+				"_assetCategoryConfigurationProvider", false);
```

Comment:
@JanisZhang Please modify the commit message, just say what you did, if there's no specific reason but we think it's safe to do so.

Reply:
@dantewang force-pushed, thanks!